### ISSUE TITLE
FP: "OffsetOperationsInspection" -> force phpdoc usage?

### ISSUE THREAD

**Issue by voku** [kalessil/phpinspectionsea#1567]
| Subject        | Details                                                                       |
| :------------- | :---------------------------------------------------------------------------- |
| Plugin         | Php Inspections (EA Ultimate), 3.0.9.1  |
| Language level | PHP 7.2                |

Current behaviour
--

Can we force the phpdoc somehow, or do I need to add ```formatOrThrowException(string $format)``` or is there a different solution for this kind of false-positives?

```php
<?php

class DateFoo {
    private int $timestamp;

    public function __construct(int $timestamp = 0) {
        $this->timestamp = $timestamp ? $timestamp : time();
    }

    /**
     * @param string $format
     *
     * @return string
     *                <p>WARNING: this method can also return false, if the $format is not correct.</p>
     */
    public function format(string $format = 'Y-m-d') {
        $return = date($format, $this->timestamp);

        if ($return === false) {
            // logging here           
        }

        return $return;
    }
}

$date = new Date();
echo $date->format(); // <-- this can't be false, because the default "format"-string is valid 
```

Extra (Live Example)
--

![example](https://user-images.githubusercontent.com/264695/82652155-6e7ea680-9c1d-11ea-846f-fe781c758acb.png)

Expected behaviour
--
no offset error

Environment details
---
PhpStorm 2020.1.1
Build #PS-201.7223.96, built on April 30, 2020
Subscription is active until February 27, 2021
Runtime version: 11.0.6+8-b765.40 amd64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
Linux 4.4.0-178-generic
GC: G1 Young Generation, G1 Old Generation
Memory: 5120M
Cores: 4
Registry: run.processes.with.pty=TRUE, ide.tooltip.initialDelay=885, ide.balloon.shadow.size=0
Non-Bundled Plugins: com.dubreuia, uk.co.ben-gibson.remote.repository.mapper, Key Promoter X, String Manipulation, com.alayouni.ansiHighlight, com.andrey4623.rainbowcsv, com.intellij.ideolog, com.upsolver.PreCommitHook, dev.arunvelsriram.desccron, mobi.hsz.idea.gitignore, name.kropp.intellij.makefile, net.seesharpsoft.intellij.plugins.csv, net.vektah.codeglance, izhangzhihao.rainbow.brackets, ru.adelf.idea.dotenv, NEON support, lv.midiana.misc.phpstorm-plugins.deep-keys, de.espend.idea.php.toolbox, de.espend.idea.php.annotation, fr.adrienbrault.idea.symfony2plugin, com.kalessil.phpStorm.phpInspectionsUltimate, me.artspb.idea.eval.plugin, net.king2500.plugins.PhpAdvancedAutoComplete, de.espend.idea.php.generics
Current Desktop: Unity
